src/store/CrosshairStore.ts

```diff
@@ -380,7 +380,8 @@ class CrosshairStore {
                         if (stx.chart.xAxis.noDraw) {
                             continue;
                         } else {
-                            fieldValue = floatDate.innerHTML;
+                            const formattedTime = floatDate.innerHTML.replace(/(\d{2})\/(\d{2})/, '$2/$1');
```

Comment:
I have no access to file with function, where the format time function is located (`chartiq`), so I added modification here (replace mm/dd ---> to dd/mm)

Reply:
@kate-deriv  let's try using 'moment' library to format the date in a way we want :)

Reply:
Will do, thank you

Reply:
I implemented solution with moment library, but found out, that format of date depends on time intervals. For ticks: it's _mm/dd hh:mm:ss_, for minutes and hours: _mm/dd hh:mm_, for days:` mm/dd/yyyy`. So in order to use moment library I need add different conditions and checks. 
What doesn't change in date is the first 5 symbols in the string: _mm/dd_. That is why usage of reg exp (which just swap first numbers) is more minimalistic solution.